src/test-util.ts

```diff
@@ -32,3 +34,195 @@ export function b(
   }
   return utf8.encode(s);
 }
+
+export function defineTestFunctions(self: Window & typeof globalThis): void {
+  // Mocha functions aren't available when importing from a new page.
+  if (typeof self.suiteSetup !== 'function') {
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    const noop = function () {};
```

Comment:
[fyi] Another way to disable this test is to add a comment (I believe)

```
const noop = () => {
  // intentionally empty
};
```

Reply:
Prefer arrow functions over function expressions. FunctionDeclaration is preferred at top level (and where required by semantics)

Reply:
You can also do `const noop = () => undefined` which would have the same semantics as an empty function

Reply:
Thanks for the `undefined` tip, I didn't know that! Searched for other non-arrow functions and didn't find any.